recipes-security/optee/optee-fiovb_git.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "OP-TEE Foundries.IO Verified Boot"
```

Comment:
Add "Client Application" since the TA is provided by the optee-os recipe itself.

Also update the commit message as this is really not optee-os related, this is just the FIOVB TEE Client application for the FIOVB TA.